### DISCUSSION TITLE
mutter-x11-scaling for GNOME 40

### DISCUSSION THREAD

**Discussion by anglepoise86** [puxplaying/mutter-x11-scaling#15] [Q&A] [answered]
Hello,

First, I would like to thank you for developing / providing us this solution to improve the HiDPI experience on Gnome.

Previously, I was on Gnome 3.38 using Arch and your solution was working fine until today, when I updated Gnome to 40 which leads to a newer version of Mutter.

So I tried to re-install mutter-x11-scaling using the manual method (git clone) but it failed because of a dependency issue, see the log below.

Before to start troubleshooting this issue, do you think that the current version mutter-x11-scaling (may) works on Gnome 40 or does it requires an update to work properly ?

Thank you in advance !

[log.txt](https://github.com/puxplaying/mutter-x11-scaling/files/6280279/log.txt)

**Comment by puxplaying** [comment]
Hi there, thanks for asking this question.
Unfortunately I can't give you the answer if the patch-set from Ubuntu will be ported to **mutter 40** for x11 fractional scaling.
The current state is that the Ubuntu patch needs to be reworked for Gnome 40. 
I will however try to invite _Marco Trevisan_ @3v1n0 to this discussion in the hope that he as Maintainer of the Ubuntu patch-set could provide more information about the future of this feature in Gnome 40.

**Reply by 3v1n0** [reply]
We decided to not upgrade to GNOME 40 this cycle in ubuntu, so as soon as next cycle will start I'll port the patches to GNOME 40 :)

**Reply by puxplaying** [reply]
Thank you for joining in and letting us know that there is hope on the horizon :-)

**Comment by puxplaying** [comment, marked as answer]
The Package has been updated to Gnome 40, please test and and report!
Big thanks @3v1n0  for porting the patches!
